File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management/Wizard.cs
namespace WizardManagement
{
  public class Wizard
  {
    private string name;
    private Broomstick broomstick;

    public string Name 
    {
      get { return name; }
      set { name = value; }
    }

    public Broomstick Broomstick
    {
      get { return broomstick; }
    }
    
    public Wizard(string name, Broomstick broomstick)
    {
      this.name = name;
      this.broomstick = broomstick;
    }

    public string Fly()
    {
      return this.broomstick.Fly();
    }
  }
}

File: week_06/day_3/composition/Composition_Start/app/src/main/java/example/codeclan/com/composition_start/wizard_management/Wizard.java
package example.codeclan.com.composition_start.wizard_management;

/**
 * Created by user on 28/08/2017.
 */

public class Wizard {
    String name;
    Broomstick broomstick;

    public Wizard(String name, Broomstick broomstick){
        this.name = name;
        this.broomstick = broomstick;
    }

    public String getName(){
        return this.name;
    }

    public Broomstick getBroomstick(){
        return this.broomstick;
    }

    public String fly(){
        return this.broomstick.fly();
    }

}
